hw/ip/keymgr/data/keymgr.hjson

```diff
@@ -175,9 +175,14 @@
       randcount: "256",
       randtype:  "data",
     },
-
-    { name: "NumInReg",
-      desc: "Number of Registers for SW inputs (SW binding / keyID / Salt)",
+    { name: "NumSaltReg",
+      desc: "Number of Registers for SW inputs (Salt)",
+      type: "int",
+      default: "4",
```

Comment:
Sorry, I have a question
from this issue #5194, @alphan said "We decided to use 256b for sw_binding and salt"
should this be 8? 8x32=256.

Reply:
ah crap. you're right. I separated out the parameters and forgot to change it to 8...